ocfweb/docs/docs/services/web/flask.md

```diff
@@ -4,81 +4,124 @@
 If you are using a group account, you may wish to consider
 [[apphosting|doc services/webapps]] instead.**
 
-Flask is a popular microframework for Python web development. Using it on the
-OCF servers requires only just a little extra configuration.
+[Flask](http://flask.pocoo.org/) is a popular microframework for Python web
+development. Using it on the OCF servers requires only just a little extra
+configuration.
 
 
-## Starting a Flask project
+## Setting up a Flask project
 
-1.    Make a directory in your ~/public_html directory:
+1. Make a new directory for your flask app in your home directory. You can name
+   this whatever you want, but we'll assume you named the directory `app`:
 
-          $ mkdir ~/public_html/flasky
+   ```shell
+   user@tsunami:~$ mkdir app
+   user@tsunami:~$ cd app
+   ```
 
-1.    Go into the directory you just created, and create two files: *.htaccess*
-      (note the dot), and *run.fcgi*.
+2. Make a virtualenv.  This will allow you to run `pip install` to install any
+   python packages that you need that the OCF does not already have. The
+   benefit of having a virtualenv is that you can decide the dependencies you
+   want, without being tied to the OCF defaults:
 
-* .htaccess:
+   ```shell
+   user@tsunami:~/app$ virtualenv -p python3 venv
```

Comment:
Thank you for suggesting people use venvs :)

Reply:
Similar to #459, perhaps `python3 -m venv venv` instead?

Reply:
As I mentioned there, on debian this apparently needs `python3-venv` to be installed, which tsunami doesn't currently have, although it sounds like it should be installed there.